### PR TITLE
Fix README.md to correct a typo

### PR DESCRIPTION
**Pull request by kmsaidev**
Fix the 66th line of the typo 'are' to 'and'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Web version is also available now. but only transmitter version. [Web version.]
 
 
 ## Contributing
-Changes are improvements are more than welcome! Feel Free to fork and open a pull request. Please make your changes in a specific branch and request to pull into `master`.
+Changes and improvements are more than welcome! Feel Free to fork and open a pull request. Please make your changes in a specific branch and request to pull into `master`.
```
Comment (designe) [inline README.md:66]: Good find!

**Review by designe — APPROVED** [review]
Thanks for contribution on README.md! :)

File: README.md (hunk 1 above)
Comment (kmsaidev) [reply README.md:66]: I'm glad you like it. I'll do my best to contribute to this project.

**Review by yeonns — APPROVED** [review]
good typo fix 👍

**Pull request merged**